unstructured/ingest/cli/cmds/qdrant.py

```diff
@@ -0,0 +1,119 @@
+import typing as t
+from dataclasses import dataclass
+
+import click
+
+from unstructured.ingest.cli.interfaces import (
+    CliConfig,
+)
+from unstructured.ingest.connector.qdrant import QdrantWriteConfig, SimpleQdrantConfig
+
+
+@dataclass
+class QdrantCliConfig(SimpleQdrantConfig, CliConfig):
+    @staticmethod
+    def get_cli_options() -> t.List[click.Option]:
+        options = [
+            click.Option(
+                ["--collection-name"],
+                required=True,
+                type=str,
+                help="The name of the Qdrant collection to use.",
+            ),
+            click.Option(
+                ["--location"],
+                type=str,
+                help="The location of the Qdrant cluster.",
+            ),
+            click.Option(
+                ["--url"],
+                type=str,
+                help="The location of the Qdrant cluster.",
+            ),
+            click.Option(
+                ["--port"],
+                type=int,
+                help="Port of the REST API interface. Default: 6333.",
+            ),
+            click.Option(
+                ["--grpc-port"],
+                type=int,
+                help="Port of the gRPC interface. Default: 6334.",
+            ),
+            click.Option(
+                ["--prefer-grpc"],
+                type=bool,
+                help="Whether to use gPRC interface whenever possible in methods. Default: False.",
+            ),
+            click.Option(
+                ["--https"],
+                type=bool,
```

Comment:
for boolean types, click also supports a `is_flag` parameter which sets it just by the presence of the cli option.  
```suggestion
                type=bool,
                is_flag=True
```